php/2020-SDK/Voice/Gather-Voice/public/index.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$BANDWIDTH_ACCOUNT_ID = getenv("BANDWIDTH_ACCOUNT_ID");
+$BANDWIDTH_API_USER = getenv("BANDWIDTH_API_USER");
+$BANDWIDTH_API_PASSWORD = getenv("BANDWIDTH_API_PASSWORD");
+$BANDWIDTH_VOICE_APPLICATION_ID = getenv("BANDWIDTH_VOICE_APPLICATION_ID");
+$BASE_URL = getenv("BASE_URL");
+$PERSONAL_NUMBER = getenv("PERSONAL_NUMBER");
+
+$config = new BandwidthLib\Configuration(
+    array(
+        "voiceBasicAuthUserName" => $BANDWIDTH_API_USER,
+        "voiceBasicAuthPassword" => $BANDWIDTH_API_PASSWORD
+    )
+);
+
+// Instantiate Bandwidth Client
+$client = new BandwidthLib\BandwidthClient($config);
+
+// Instantiate App
+$app = AppFactory::create();
+
+// Add error middleware
+$app->addErrorMiddleware(true, true, true);
+
+$voice_client = $client->getVoice()->getClient();
+
+$app->post('/outboundCall', function (Request $request, Response $response) {
+    // POST with to, from, and tag creates outbound call
+    global $BANDWIDTH_ACCOUNT_ID, $BANDWIDTH_VOICE_APPLICATION_ID, $BASE_URL, $voice_client;
+    $data = $request->getParsedBody();
+    $body = new BandwidthLib\Voice\Models\ApiCreateCallRequest();
+    $body->from = $data['from'];
+    $body->to = $data['to'];
+    $body->answerUrl = $BASE_URL . "/voiceCallback";;
+    $body->applicationId = $BANDWIDTH_VOICE_APPLICATION_ID;
+
+    try {
+        $apiResponse = $voice_client->createCall($BANDWIDTH_ACCOUNT_ID, $body);
+        $callId = $apiResponse->getResult()->callId;
+        $response->getBody()->write('{"callId": "'.$callId.'"}');
+        return $response->withStatus(201)
+          ->withHeader('Content-Type', 'application/json');
+    } catch (BandwidthLib\APIException $e) {
+        print_r($e);
+    }
+    return $response->withStatus(201);
```

Comment:
Do we want to do a different return status if something goes wrong? Not sure how in/out of scope that would be though

Reply:
Good catch - started with a blanket 201 and then forgot to build out the response if an error was encountered. Fixed to return a 400 and the error in the body